crates/tauri-bundler/src/bundle/windows/msi/update-task.xml

```diff
@@ -37,7 +37,7 @@
   <Actions Context="Author">
     <Exec>
       <Command>cmd.exe</Command>
-      <Arguments>/c "%SYSTEMROOT%\System32\msiexec.exe /i %TEMP%\\{{product_name}}.msi {{msiexec_args}} /promptrestart"</Arguments>
+      <Arguments>/c ^"%SYSTEMROOT%\System32\msiexec.exe /i "%TEMP%\\{{product_name}}.msi" {{msiexec_args}} /promptrestart^"</Arguments>
```

Comment:
why are we escaping the first quotes here, can't we use a single quote `'` instead?

Reply:
In windows single and double quotes dont work similar to mac or linux. 

in linux, you would put one kind of quotes in another but to achieve similar result in windows, second kind of quotes is `^"` . This is not escaping quotes, its second type of quotes that wont conflict with regular ones

[source: ](https://ss64.com/nt/syntax-esc.html#:~:text=When%20ECHOing%20a%20string%20with%20an%20odd%20number%20(or%20just%20an%20unknown%20number)%20of%20embedded%20quotes%20through%20the%20pipeline%2C%20the%20outer%20quotes%20should%20be%20escaped%3A)

Reply:
![image](https://github.com/user-attachments/assets/59b9f294-12f9-4383-b1cf-0b5e080e6675)

Reply:
Just added change message. Let me know if that works

Reply:
thanks for clarification